api/backup/restore.go

```diff
@@ -42,7 +42,8 @@ func RestoreArchive(archive io.Reader, password string, filestorePath string, ga
 		return errors.Wrap(err, "Failed to stop db")
 	}
 
-	if err = restoreFiles(restorePath, filestorePath); err != nil {
+	dbFile := datastore.Connection().GetDatabaseFileName()
```

Comment:
would it be a problem if we have unencrypted file in the archive, while current db is encrypted?

Reply:
It'll migrate.   And if there is no key it won't start.  Which is expected.

Reply:
👍